apps/admin/backend/schema.sql

```diff
@@ -42,13 +42,23 @@ create table cvrs (
   ballot_type text not null 
     check (ballot_type = 'absentee' or ballot_type = 'precinct' or ballot_type = 'provisional'),
   batch_id text not null,
-  batch_label text not null,
   party_id text,
   precinct_id text not null,
   scanner_id text not null,
   sheet_number integer check (sheet_number is null or sheet_number > 0),
   votes text not null,
   created_at timestamp not null default current_timestamp,
+  foreign key (election_id) references elections(id)
+    on delete cascade,
+  foreign key (batch_id, scanner_id) references scanner_batches(batch_id, scanner_id)
+);
```

Comment:
nit: The index creation statements for `cvrs` aren't co-located with the table anymore

Reply:
good nit, d517c0085